cloud-azure/src/main/java/com/sequenceiq/cloudbreak/cloud/azure/ResourceGroupUsage.java

```diff
@@ -2,5 +2,6 @@
 
 public enum ResourceGroupUsage {
     SINGLE,
-    MULTIPLE
+    MULTIPLE,
+    SINGLE_WITH_DEDICATED_STORAGE_ACCOUNT
```

Comment:
I would add a method as isSingleResourceGroup(), as in some cases the flavor of the single RG is not important.